src/main/java/org/schedulesdirect/api/Airing.java

```diff
@@ -267,9 +273,9 @@
 			timeApproximate = src.optBoolean("timeApproximate");
 			if(subtitled && src.has("subtitledLanguage"))
 				subtitleLanguage = src.getString("subtitledLanguage");
-			sap = src.optBoolean("sap");
+			/*sap = src.optBoolean("sap");
```

Comment:
Was sapLanguage removed from the json?  If so, I'd just prefer to remove dead code instead of commenting it out.